sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java

```diff
@@ -89,6 +92,73 @@ void setup() {
         this.soapUtil = riInjector.getInstance(SoapUtil.class);
     }
 
+    @Test
+    @DisplayName("A SERVICE PROVIDER SHALL include the parent descriptor handle in "
+            + "msg:DescriptionModificationReport/msg:ReportPart/@ParentDescriptor for any Descriptor that is not a "
+            + "pm:MdsDescriptor, if msg:DescriptionModificationReport/msg:ReportPart/@ModificationType "
+            + "is “Crt” (Created).")
+    @TestIdentifier(EnabledTestConfig.BICEPS_R0055_0)
+    @TestDescription("Retrieves all DescriptionModificationReports seen during the TestRun and checks for each "
+            + "created AbstractDescriptor which is not an MdsDescriptor that the attribute @ParentDescriptor "
+            + "of its ReportPart is set.")
+    @RequirePrecondition(simplePreconditions = ConditionalPreconditions.DescriptionModificationCrtPrecondition.class)
+    void testRequirementR00550() throws NoTestData, IOException {
+        final var mdibHistorian = mdibHistorianFactory.createMdibHistorian(
+                messageStorage, getInjector().getInstance(TestRunObserver.class));
+
+        final var acceptableSequenceSeen = new AtomicInteger(0);
+
+        try (final Stream<String> sequenceIds = mdibHistorian.getKnownSequenceIds()) {
+            sequenceIds.forEach(sequenceId -> {
+
+                // get DescriptionModification reports
+                try (final var reports = messageStorage.getInboundMessagesByBodyTypeAndSequenceId(
```

Comment:
Use _getInboundMessagesByBodyType_ instead, you don't need the sequence ids and can remove the mdibHistorian.

Reply:
Indeed. Done.